packages/webview_flutter/webview_flutter_wkwebview/darwin/Classes/FWFScrollViewHostApi.m

```diff
@@ -24,47 +24,65 @@ - (instancetype)initWithInstanceManager:(FWFInstanceManager *)instanceManager {
   return self;
 }
 
+#if TARGET_OS_IOS
 - (UIScrollView *)scrollViewForIdentifier:(NSInteger)identifier {
   return (UIScrollView *)[self.instanceManager instanceForIdentifier:identifier];
 }
+#endif
 
 - (void)createFromWebViewWithIdentifier:(NSInteger)identifier
                       webViewIdentifier:(NSInteger)webViewIdentifier
                                   error:(FlutterError *_Nullable __autoreleasing *_Nonnull)error {
+#if TARGET_OS_IOS
   WKWebView *webView = (WKWebView *)[self.instanceManager instanceForIdentifier:webViewIdentifier];
   [self.instanceManager addDartCreatedInstance:webView.scrollView withIdentifier:identifier];
+#else
+  *error = [FlutterError errorWithCode:@"UnavailableApi"
+                               message:@"scrollView is unavailable on macOS"
+                               details:nil];
+#endif
 }
 
 - (NSArray<NSNumber *> *)
     contentOffsetForScrollViewWithIdentifier:(NSInteger)identifier
                                        error:(FlutterError *_Nullable *_Nonnull)error {
+#if TARGET_OS_IOS
   CGPoint point = [[self scrollViewForIdentifier:identifier] contentOffset];
   return @[ @(point.x), @(point.y) ];
+#else
+  return @[ @(0), @(0) ];
```

Comment:
will this function be called by macOS? if not, maybe use `API_AVAILABLE` annotation? same below